deploy/Dockerfile

```diff
@@ -1,28 +1,39 @@
-FROM golang:alpine as builder
+FROM docker.io/library/golang:1.14-alpine as builder
+
 MAINTAINER Jack Murdock <jack_murdock@comcast.com>
 
 WORKDIR /go/src/github.com/xmidt-org/tr1d1um
+
 ARG VERSION=unknown
 ARG GITCOMMIT=unknown
 ARG BUILDTIME=unknown
 
-RUN apk update && apk upgrade && \
-    apk add --no-cache bash git openssh
+
+RUN apk add --no-cache --no-progress \
+    ca-certificates \
+    make \
+    git \
+    openssh \
+    gcc \
+    libc-dev \
+    upx
 
 COPY . .
-RUN GO111MODULE=on go build \
-  -o tr1d1um_linux_amd64 \
-  -ldflags "-X 'main.BuildTime=${BUILDTIME}' -X main.GitCommit=${GITCOMMIT} -X main.Version=${VERSION}"
+RUN make build
+
+FROM scratch
 
-FROM alpine
+COPY --from=builder /etc/passwd /etc/passwd
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=builder /go/src/github.com/xmidt-org/tr1d1um/tr1d1um.yaml /tr1d1um.yaml
+COPY --from=builder /go/src/github.com/xmidt-org/tr1d1um/tr1d1um /tr1d1um
+COPY --from=builder /go/src/github.com/xmidt-org/tr1d1um/deploy/Dockerfile /go/src/github.com/xmidt-org/tr1d1um/NOTICE /go/src/github.com/xmidt-org/tr1d1um/LICENSE /go/src/github.com/xmidt-org/tr1d1um/CHANGELOG.md /
 
-RUN apk --no-cache add ca-certificates
-RUN mkdir -p /etc/tr1d1um
-VOLUME /etc/tr1d1um
+EXPOSE 6100
+EXPOSE 6101
+EXPOSE 6102
+EXPOSE 6103
 
-EXPOSE 8080
+USER nobody
```

Comment:
do we need this command? If so, maybe `xmidt` would be a better name and we keep it the same across all Makefiles